Add Actionsflow to Automation

theowenyoung opened the pull request:
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] Format your submission as follows, where `Demo` and `Clients` are optional.
  Do not add a duplicate `Source code` link if it is the same as the main link.
  Keep the short description under 250 characters and use [sentence case](https://en.wikipedia.org/wiki/Letter_case#Sentence_case)
  for it, even if the project's webpage or readme uses another capitalisation
  such as title case, all caps, small caps or all lowercase.
  ``- [Name](http://homepage/) - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that depend on proprietary services outside the user's control must be marked `⚠`.
  ``- [Name](http://homepage/) `⚠` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `License` `Language` ``
- [x] Additions that are not [Free software](https://en.wikipedia.org/wiki/Free_software)
  must be added to `non-free.md` and marked `⊘ Proprietary`:
  ``- [Name](http://homepage/) `⊘ Proprietary` - Short description, under 250 characters, sentence case. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code), [Clients](https://url.to/list/of/related/clients-or-apps)) `Language` ``
- [x] Additions are inserted preserving alphabetical order.
- [x] Additions are not already listed at any of
  - [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin) (IT infrastructure management),
  - [awesome-analytics](https://github.com/onurakpolat/awesome-analytics) (analytics),
  - [staticgen.com](https://www.staticgen.com/)
  - [staticsitegenerators.net](https://staticsitegenerators.net/) (static site generators).
- [x] The `Language` tag is the main server-side requirement for the software - don't include frameworks or specific dialects.
- [x] Any license you add is in our [list of licenses](https://github.com/awesome-selfhosted/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted/pulls), including closed ones.
- [x] Any category you are creating has the minimum requirement of 3 items.
  If not, your addition may be inserted into `Misc/Other`.
- [x] Any software project you are adding to the list is actively maintained.
- [x] The pull request title is informative, unlike "Update README.md".
  Suggested titles: "Add aaa to bbb" for adding software aaa to section bbb,
  "Remove aaa from bbb" for removing, "Fix license for aaa", etc.

nodiscc reviewed (changes requested):
- Please change the description to  `The free Zapier/IFTTT alternative for developers to automate your workflows based on Github actions` (original project description)
- Please add the `⚠`  symbol (it depends on Github Actions/SaaS)

theowenyoung commented:
Thanks, I've changed that.

nodiscc reviewed (changes requested):
Thanks.

Please change the language/platform to `Docker/Nodejs` (Docker is the only supported [installation method](https://actionsflow.github.io/docs/self-hosted/)

> docker run -it -v /var/run/docker.sock:/var/run/docker.sock

Why does it need r/w access to the docker socket? This is a security risk (the application effectively has root access to the host)

theowenyoung commented:
@nodiscc Thanks for the review.

I have changed the language to `Docker/Nodejs`.

For self-hosted version, Actionsflow's dependency [act](https://github.com/nektos/act) need `/var/run/docker.sock` to run and manage the Github Actions. So we have to give it the root access. I have added a notice at Actionsflow docs.

https://actionsflow.github.io/docs/self-hosted/
